<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exchange_rate</class>
 <widget class="QWidget" name="exchange_rate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="origin_edit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_equal">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>  =</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="transmit_unit">
       <item>
        <property name="text">
         <string>人民币(CNY)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(原神)原石</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(CFM)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(LOL)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(王者荣耀)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>美丽币(USD)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>坤</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_calculate">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>汇率计算</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="transmit">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="origin_unit">
       <item>
        <property name="text">
         <string>人民币(CNY)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(原神)原石</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(CFM)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(LOL)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(王者荣耀)点卷</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>美丽币(USD)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>坤</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="transmit_edit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
